pytorch_pfn_extras/training/extensions/_snapshot.py

```diff
@@ -348,9 +348,9 @@ def initialize(  # type: ignore[override]
             # terms of mtime, and tries to load it it the target or
             # manager.
             assert writer is not None
-            loaded_fn = _find_latest_snapshot(self.filename, outdir, writer.fs)
+            loaded_fn = _find_latest_snapshot(self.filename, writer.out_dir, writer.fs)
```

Comment:
Are we no longer searching in `outdir`? (`manager.out`). Also, we might want to update the comment above this line accordingly.

Reply:
Fixed it.